indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,134 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        @include border-left($width: 3px);
+        padding-left: 10px;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+        .row-title {
+            @include border-bottom();
+            font-size: 14pt;
+        }
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.review-role {
+                @include border-all($gray);
+                margin-right: 10px;
+                background-color: white;
+                color: $light-black;
+                i {
+                    color: $blue;
+                }
+            }
+
+            &.reviews {
+                margin-top: 0;
+                border-width: 1px;
+                border-style: solid;
+                //border: 1px solid;   not used not useed
+                border-bottom-right-radius: 0;
+                border-top-right-radius: 0;
+            }
+            .icon {
+                //    not used
+                display: inline-block;
+                margin-right: 7px;
+            }
+
+            &.submitted {
+                background-color: $blue;
+            }
+
+            &.accepted {
+                background-color: $green;
+            }
+
+            &.rejected {
+                background-color: $red;
+            }
+
+            &.to_be_corrected {
+                background-color: $yellow;
+            }
+
+        }
+        .paper-list ul {
+            list-style-type: none;
+            padding-left: 0;
+        }
+
+        .row-content {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        .row-right a {
+            justify-content: flex-end;
+        }
+    }
+
+    ul {
+        list-style-type: none;
+        padding-left: 0;
+    }
+}
+
+.call-for-papers {
+    .cfp-infoline-section {
+        .infoline {
+            float: none;
+        }
+    }
+    .cfp-announcement {
+        color: $light-black;
+        padding: 20px 0 10px;
+    }
+}
+
+.display-contribution-list ul {
+    list-style-type: none;
+    padding-left: 0;
+}
+
+.deadline-info {
+    @include border-all($blue);
```

Comment:
we never really agreed on something, I like the empty line, @OmeGak doesn't. So there you go!